openc3/lib/openc3/api/cmd_api.rb

```diff
@@ -173,7 +177,7 @@ def send_raw(interface_name, data, scope: $openc3_scope, token: $openc3_token)
     # @param target_name [String] Target name of the command
     # @param command_name [String] Packet name of the command
     # @return [Hash] command hash with last command buffer
-    def get_cmd_buffer(*args, scope: $openc3_scope, token: $openc3_token)
+    def get_cmd_buffer(*args, manual: false, scope: $openc3_scope, token: $openc3_token)
       target_name, command_name = _extract_target_command_names('get_cmd_buffer', *args)
       authorize(permission: 'cmd_info', target_name: target_name, packet_name: command_name, scope: scope, token: token)
```

Comment:
Any harm in passing manual down to authorize always?

Reply:
I moved the logic all into the Enterprise authorize method like we talked about